docs/uppmax.md

```diff
@@ -74,14 +76,170 @@ It is not suitable for use with real data.
 
 To use it, submit with `-profile uppmax,devel`.
 
-## Running on Bianca
+## Running on bianca
+
+> :warning: For more information about `bianca`, follow the [UPPMAX `bianca` user guide](http://uppmax.uu.se/support/user-guides/bianca-user-guide/).
+> :warning: For more information, follow the [nf-core guide for running offline](https://nf-co.re/usage/offline) and the [nf-core `tools` guide for downloading pipelines for offline use](https://nf-co.re/tools#downloading-pipelines-for-offline-use).
```

Comment:
Could you add bullets here please? I suspect that when we finally get these to render on the website it'll all go into one long line of text (GitHub renders single newlines but the website doesn't)